moj-node/server/views/pages/search.html

```diff
@@ -61,4 +61,13 @@ <h3 class="govuk-heading-m">No results found for "{{ query }}"</h3>
     </div>
   </div>
 
+  <div class="govuk-width-container govuk-hub-article-feedback govuk-!-margin-top-9">
+    <p class="govuk-body">This is an early version of search, tell us what you think:</p>
+    {{ hubFeedbackWidget({
+      title: "Search - " + query,
```

Comment:
Assuming query isn't directly coming from the querystring? @romidane @LucasCairns ?

Reply:
No it's coming from the server

Reply:
But it's worth us making sure that express sanitizes the query strings